app/Jobs/Emails/PresentationSubmissions/SelectionProcess/PresentationSubmitterSelectionProcessEmail.php

```diff
@@ -0,0 +1,81 @@
+<?php namespace App\Jobs\Emails\PresentationSubmissions\SelectionProcess;
+/**
+ * Copyright 2023 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+use App\Jobs\Emails\AbstractEmailJob;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Log;
+use models\main\Member;
+use models\summit\Summit;
+use utils\Filter;
+
+/**
+ * Class PresentationSubmitterSelectionProcessEmail
+ * @package App\Jobs\Emails\PresentationSubmissions
+ */
+abstract class PresentationSubmitterSelectionProcessEmail extends AbstractEmailJob
+{
+    protected $filter;
+
+    /**
+     * @param Summit $summit
+     * @param Member $submitter
+     * @param string|null $test_email_recipient
+     * @param Filter|null $filter
+     */
+    public function __construct
+    (
+        Summit $summit,
+        Member $submitter,
+        ?string $test_email_recipient,
+        ?Filter $filter = null
+    ){
+
+        if(!is_null($filter))
+            $this->filter = $filter->getOriginalExp();
+        $payload  = [];
+
+        $payload['summit_name'] = $summit->getName();
+        $payload['summit_logo'] = $summit->getLogoUrl();
+        $payload['summit_schedule_url'] = $summit->getScheduleDefaultPageUrl();
+        $payload['summit_site_url'] = $summit->getLink();
+        $payload['submitter_full_name'] = $submitter->getFullName();
```

Comment:
u need to provide here the 
following collections
* accepted_presentations
* alternate_presentations
* rejected_presentations

Reply:
done